resources/assets/less/bem/user-list.less

```diff
@@ -0,0 +1,32 @@
+/**
+ *    Copyright 2015-2018 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+.user-list {
+  align-self: flex-start;
+  flex-grow: 1;
+
+  &__content {
+    max-width: 250px;
+    overflow: hidden;
+    height: auto;
+
+    &--folded {
+      height: 0px;
```

Comment:
Why not just `display: none` so the `overflow: hidden` isn't needed? And pretty sure `height: auto` isn't needed in any case.

Also maybe want to check jquery's `slideToggle` if that's the original intention.